libs/icons/scripts/optimize-svgs.ts

```diff
@@ -18,29 +18,44 @@ const libraryName = 'sage-icons';
  *
  * @param rootDir - The source directory
  */
-const build = async(rootDir: string) => {
+export const run = async(rootDir: string, optimizeFiles = false) => {
```

Comment:
The parameter `optimizeFiles` is used to determine if the `figmaIconProcess` has been run. It also determines the "source" path of the SVGs to use for package generation.